### PR TITLE
Added Core Chain Network (ChainID: 1116 ) to networks.json

### PR DESCRIPTION
**Pull request by leeyanbtcs**
tested on https://snapshot-networks.on.fleek.co/1116
last block fetched correctly
no error messages

image

### CODE REVIEW

**Comment by leeyanbtcs** [comment]
@bonustrack can review and approve the PR?

**Comment by leeyanbtcs** [comment]
@bonustrack  can you review and approve the PR?

File: src/networks.json [hunk 1]
```diff
@@ -2190,5 +2190,21 @@
...
+  },
+  "1116": {
+    "key": "1116",
+    "name": "Core Chain Mainnet",
+    "shortName": "Core",
+    "chainId": 1116,
+    "network": "mainnet",
+    "multicall": "0x024f0041b76B598c2A0a75004F8447FaF67BD004",
+    "rpc": [
+      "https://rpceuwest3.coredao.org"
```
Comment (ChaituVR) [inline src/networks.json:2202]: Hi @leeyanbtcs Looks like this is not a full archive node? i get `missing trie node a705014fcbd00e291767bda7479b41ecdd9b5b20604d061e19acb01b95a9153d` error

**Comment by ChaituVR** [comment]
Hey @leeyanbtcs we already support this network

**Pull request closed without merging**